devchat.go

```diff
@@ -183,7 +184,7 @@ type user struct {
 	session       ssh.Session
 	term          *terminal.Terminal
 	bell          bool
-	color         color.Color
+	color         string
```

Comment:
I don't really understand the motivation behind changing this to a string, what did you have in mind?

Reply:
The point is, to support colors beyond single colored strings. Other colors would still be possible with `u.color` being a `color.Color`, but we can not store the state of those special colors then!

Reply:
ah so you want it to be able to be serialized?

Reply:
Yes! Absolutely. We may need this later, right? And as rainbow is not storable in `color.Color`. Afaik a string or an enum would be the way to go... Do you have another way?

Reply:
Yep that looks pretty nice to me (separating the two changes into two prs would have been nice but this is fine! we're gonna serialize more data (such as bell settings, timezone) later on anyway)